src/main.cpp

```diff
@@ -2,6 +2,6 @@
 
 int main()
 {
-	std::cout << "Hello World!" << std::endl;
-	return EXIT_SUCCESS;
```

Comment:
I see you already merged it into controller PR, so please close this PR.
Keep in mind that this creates one big PR that contains many activities instead of having dedicated PR for each activity, please use an integration/dev branch if you don't want to commit to main yet